modules/nvidia_plugin/src/CMakeLists.txt

```diff
@@ -37,11 +37,7 @@ ie_add_plugin(NAME ${TARGET_NAME}
 # Enable support of CC for the plugin
 ie_mark_target_as_cc(${TARGET_NAME})
 
-if (${CUDAToolkit_VERSION_MAJOR} LESS_EQUAL "10")
-    set_property(TARGET ${OBJ_NAME} PROPERTY CUDA_ARCHITECTURES 30 35 50 60 72)
-else()
-    set_property(TARGET ${OBJ_NAME} PROPERTY CUDA_ARCHITECTURES 35 50 60 72)
-endif()
+set_property(TARGET ${OBJ_NAME} PROPERTY CUDA_ARCHITECTURES ${CMAKE_CUDA_ARCHITECTURES})
```

Comment:
Can we add printing of `CMAKE_CUDA_ARCHITECTURES` just to make it visible in cmake logs?

Reply:
Done (in modules/nvidia_plugin/CMakeLists.txt)